### PR TITLE
Feat/CTA-12_Get_QuoteRequest_By_Id

### PR DESCRIPTION
**Pull request by dazhuox**
JIRA: [link to jira ticket](https://champlainsaintlambert.atlassian.net/browse/CTA-12?atlOrigin=eyJpIjoiMTI2ZTljYTQwZGFlNDVjYmJiNDdkNjVkZjExZDFjYzYiLCJwIjoiaiJ9)
## Context:
This implements the crud operation of getting a quoteRequest by Id and presents in a nice-looking fashion.
## Changes
Changed: 
- QuoteRequestService
- QuoteRequestServiceImpl
- QuoteRequestController
- QuoteRequestRepository
- QuoteRequestServiceImplUnitTest
- getClientProfile.cy.js
- QuotesAdmin.jsx
- QuotesClient.jsx
- QuotesAdmin.css
## Before and After UI (Required for UI-impacting PRs)
Before:
image

After:
image

### CODE REVIEW

File: coupetonarbrefrontend/src/pages/QuotesAdmin.jsx [hunk 1]
```diff
@@ -40,7 +60,7 @@ function QuotesAdmin() {
       .replace(/_/g, ' ')
       .replace(/[^a-zA-Z ]/g, '')
       .toLowerCase()
-      .replace(/\b\w/g, (match) => match.toUpperCase());
+      .replace(/\b\w/g, (l) => l.toUpperCase());
```
Comment (Matthewtan9) [inline coupetonarbrefrontend/src/pages/QuotesAdmin.jsx:63]: Why the change from match to I.

**Review by Matthewtan9 — APPROVED** [review]
Good job!

**Review by drallo22 — APPROVED** [review]
Good work!
